src/vs/platform/telemetry/test/browser/telemetryService.test.ts

```diff
@@ -96,156 +95,148 @@ suite('TelemetryService', () => {
 
-			service.dispose();
-		});
+		service.dispose();
 	});
 
 	test('TelemetryInfo comes from properties', function () {
 		const service = new TelemetryService({
 			appenders: [NullAppender],
-			commonProperties: Promise.resolve({
+			commonProperties: {
 				sessionID: 'one',
 				['common.machineId']: 'three',
-			})
+			}
 		}, new TestConfigurationService(), TestProductService);
 
-		return service.getTelemetryInfo().then(info => {
-			assert.strictEqual(info.sessionId, 'one');
-			assert.strictEqual(info.machineId, 'three');
+		assert.strictEqual(service.sessionId, 'one');
+		assert.strictEqual(service.machineId, 'three');
 
-			service.dispose();
-		});
+		service.dispose();
 	});
 
-	test('telemetry on by default', sinonTestFn(function () {
+	test('telemetry on by default', function () {
 		const testAppender = new TestTelemetryAppender();
 		const service = new TelemetryService({ appenders: [testAppender] }, new TestConfigurationService(), TestProductService);
 
-		return service.publicLog('testEvent').then(() => {
-			assert.strictEqual(testAppender.getEventsCount(), 1);
-			assert.strictEqual(testAppender.events[0].eventName, 'testEvent');
+		service.publicLog('testEvent');
+		assert.strictEqual(testAppender.getEventsCount(), 1);
+		assert.strictEqual(testAppender.events[0].eventName, 'testEvent');
 
-			service.dispose();
-		});
-	}));
+		service.dispose();
+	});
 
 	class JoinableTelemetryService extends TelemetryService {
 
-		private promises: Promise<void>[] = [];
+		// private promises: Promise<void>[] = [];
 
 		constructor(config: ITelemetryServiceConfig) {
 			super({ ...config, sendErrorTelemetry: true }, new TestConfigurationService, TestProductService);
-			this.promises = this.promises ?? [];
-			this.promises = this.promises ?? [];
-		}
-
-		join(): Promise<any> {
-			return Promise.all(this.promises);
-		}
-
-		override publicLog(eventName: string, data?: ITelemetryData): Promise<void> {
-			const p = super.publicLog(eventName, data);
-			// publicLog is called from the ctor and therefore promises can be undefined
-			this.promises = this.promises ?? [];
-			this.promises.push(p);
-			return p;
+			// this.promises = this.promises ?? [];
+			// this.promises = this.promises ?? [];
 		}
 
-		override publicLogError(errorEventName: string, data?: ITelemetryData): Promise<void> {
-			const p = super.publicLogError(errorEventName, data);
-			// publicLogError is called from the ctor and therefore promises can be undefined
-			this.promises = this.promises ?? [];
-			this.promises.push(p);
-			return p;
-		}
-
-		override publicLogError2<E extends ClassifiedEvent<OmitMetadata<T>> = never, T extends IGDPRProperty = never>(eventName: string, data?: StrictPropertyCheck<T, E>): Promise<any> {
-			return this.publicLogError(eventName, data as ITelemetryData);
-		}
+		// join(): Promise<any> {
+		// 	return Promise.all(this.promises);
+		// }
+
+		// override publicLog(eventName: string, data?: ITelemetryData): Promise<void> {
```

Comment:
What's with all this commented out tests?

Reply:
Forgot to actually delete all this unnecessary code.